static/index.js

```diff
@@ -0,0 +1,143 @@
+const REVIEWS_LOCAL_STORAGE_KEY = "reviews";
+let reviews = getReviewsFromLocalStorage();
+
+function handleFormSubmit(event) {
+  event.preventDefault();
+
+  const { description, rating } = event.target.elements;
+  const newReview = {
```

Comment:
Za `description` imas required atribut i to je fine, ali ti nedostaje guard za `rating`. 
Možeš submitati review bez rating-a.